tests/sdk/test_list.py

```diff
@@ -71,5 +71,5 @@ def test_list_asset_with_filters_failure(client, mocker):
     with pytest.raises(ValueError) as exc_info:
         client.list_algo(filters)
 
-    assert m.is_called()
+    m.assert_not_called()
```

Comment:
Here, `m` should not be called, contrary to what was specified before 🙂

Reply:
👍

Reply:
👍